threat_model.md

```diff
@@ -0,0 +1,190 @@
+
+*Impact:* Without *Tanker*, the attacker immediately gains access to all user data existing at this point in time. By repeating the same attack, they also have access to all future user data. With *Tanker*, they have no access to Tanker-encrypted data. 
+
+**App secret + repeatable data access**:
+
+*Attack*: An attacker gains access to all user data stored by the *application*. They also gain access to the private [Trustchain Signature Key Pair] contained in the app secret. This attack is repeatable. 
+
+*Impact:* Without *Tanker*, the attacker immediately gains access to all user data existing at this point in time. By repeating the same attack, they also have access to all future user data. With *Tanker*, they have no access to Tanker-encrypted data. 
+
+**Identity modification + one-time data access** :
+
+*Attack*: An attacker gains access to all user data stored by the *application*. They also gain access to all stored *Tanker Identities* and are able to change them. They replace all *user* *Identities* by their own. This attack is not repeatable. 
+
+*Impact:* Without *Tanker*, the attacker immediately gains access to all user data existing at this point in time. With *Tanker*, they have no access to Tanker-encrypted data. All future user data will however be encrypted for the attacker's *Tanker* account.
+
+**Identity modification + repeatable data access** :
+
+*Attack*: An attacker gains access to all user data stored by the *application*. They also gain access to all stored *Tanker Identities* and are able to change them. This attack is repeatable. 
+
+*Impact:* Without *Tanker*, the attacker immediately gains access to all user data existing at this point in time. With *Tanker*, all future user data will be encrypted for the attacker's *Tanker* account. If the attacker accesses the data again in the future, they will gain access to all data since the initial attack.
+
+**App code control**:   
+
+*Attack*: An attacker is able to deploy arbitrary code instead of the *application*. They deploy a version of the *application* which sends all user data in clear to a server they control.
+
+*Impact:* Whatever the level of protection, the attacker has access to all past and future data. 
+
+*Fix:* *Application* verification and signature could help mitigate that risk. This technology however is difficult to implement in browsers.
+
+## User risks factors
+
+| Threat                  | No Tanker                               | Tanker                                  |
+| ----------------------- | --------------------------------------- | --------------------------------------- |
+| Unlocked device         | Targeted access to past and future data | Targeted access to past and future data |
+| Locked device           | Targeted access to past data            | No access                               |
+| Password / Email access | Targeted access to past and future data | Targeted access to past and future data |
```

Comment:
This is more `Locked device + user app credentials` because `No device + user app credentials` does not give access to data without a successful Tanker Identity Verification.